### PR TITLE
[#607] Added S3_BUCKET_NAME env var to Airflow Worker and Jenskins Slave

### LINKED ISSUE #607: Jenkins slave and Airflow worker don't know S3 bucket name for a cluster
Currently It's possible to connect to S3 using kube2iam from Jenkins slave and AIrflow Worker. But it's not possible to get S3 bucket name that is defined in *legion_data_s3_bucket* profile property.

### PR DESCRIPTION
**Pull request by aliaksandr-d**
This PR closes #607 
I've passed cluster s3 bucket name from profile file as S3_BUCKET_NAME environment variable to Airflow Worker and Jenkins Slave.

### CODE REVIEW

**Comment by aliaksandr-d** [comment]
bot, test all

**Comment by aliaksandr-d** [comment]
bot, test all

**Comment by aliaksandr-d** [comment]
*Check if flower scale down works properly* test failed:
`Expected ready workers count '1' does not match actual '2'`

File: deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2 [hunk 1]
```diff
@@ -18,6 +18,9 @@ secrets:
   fernet_key: {{ airflow.fernet_key }}
   flask: {{ secret_flask }}
 
+  s3:
```
Comment (kirillmakhonin) [inline deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2]: What is this? Looks obsolete

Comment (aliaksandr-d) [reply deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2]: Fixed

**Comment by aliaksandr-d** [comment]
bot, test all

**Comment by aliaksandr-d** [comment]
`*Check EDI scale up all instances for 2 models(diff versions) by versions=*` test failed with a message:

`2 != 0`

**Pull request merged**